tests/test_systems/test_spectrometers/test_HyperspectralImager.py

```diff
@@ -123,27 +134,33 @@ def test_get_optical_spectral_resolution():
 
 
 def test_get_sensor_spectral_resolution():
-    """Test get_sensor_spectral_resolution."""
-
+    """Test of get_sensor_spectral_resolution."""
```

Comment:
```suggestion
    """Test get_sensor_spectral_resolution."""
```
Remove 'of' for consistency